src/main/java/org/opentripplanner/util/geometry/GeometryUtils.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.common.geometry;
+package org.opentripplanner.util.geometry;
```

Comment:
We might want to move `convertGeoJsonToJtsGeometry` and associated functions into a mapper class

Reply:
added todo